sktime/datatypes/_series/_convert.py

```diff
@@ -172,3 +174,67 @@ def convert_np_to_UvS_as_Series(obj: np.ndarray, store=None) -> pd.Series:
 
 
 convert_dict[("np.ndarray", "pd.Series", "Series")] = convert_np_to_UvS_as_Series
+
+
+# obtain other conversions from/to numpyflat via concatenation to numpy3D
+def _concat(fun1, fun2):
+    def concat_fun(obj, store=None):
+        obj1 = fun1(obj, store=store)
+        obj2 = fun2(obj1, store=store)
+        return obj2
+
+    return concat_fun
+
+
+def _extend_conversions(mtype, anchor_mtype, convert_dict):
```

Comment:
If datatypes is refactored, then this method should be placed somewhere else. Currently similar methods are implemented in panels and tables.

Reply:
yes, indeed - too much copy-paste!
Though I would go with incurring technical debt due to priorities...

Re refactor directions:

@chrisholder started sth, but abandoned it.
https://github.com/alan-turing-institute/sktime/pull/1460
Haven´t had the time to salvage it.

This would be another option:
https://github.com/dylan-profiler/visions/issues/194